9c-main/chart/templates/remote-action-evaluator-headless.yaml

```diff
@@ -0,0 +1,245 @@
+        name: probe-script
+      - name: download-snapshot-script
+        configMap:
+          defaultMode: 0700
+          name: {{ $.Chart.Name }}-download-snapshot-script
+      - name: remote-action-evaluator-headless-configmap
+        configMap:
+          defaultMode: 0700
+          name: {{ $.Chart.Name }}-remote-action-evaluator-headless-configmap
+      {{- if $.Values.remoteActionEvaluatorHeadless.loggingEnabled }}
+      - hostPath:
+          path: /var/log/headless
+          type: DirectoryOrCreate
+        name: json-log
+      {{- end }}
+  updateStrategy:
+    type: RollingUpdate
+  volumeClaimTemplates:
+  - metadata:
+      name: remote-action-evaluator-headless-data
+    spec:
+      accessModes:
+      - ReadWriteOnce
+      resources:
+        requests:
+          storage: {{ $.Values.remoteActionEvaluatorHeadless.storage.data }}
+      storageClassName: {{ $.Chart.Name }}-gp3
+      volumeMode: Filesystem
+
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: lib9c-state-service
+    app.kubernetes.io/instance: {{ $.Chart.Name }}
+  name: lib9c-state-service
+  namespace: {{ $.Chart.Name }}
+spec:
+  replicas: 1
+  revisionHistoryLimit: 3
+  selector:
+    matchLabels:
+      app: lib9c-state-service
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        app: lib9c-state-service
+      name: lib9c-state-service
+    spec:
+      containers:
+      - args:
+        - Lib9c.StateService.dll
+        - --urls=http://0.0.0.0:5157/
+        command:
+        - dotnet
+        image: "{{ $.Values.lib9cStateService.image.repository }}:{{ $.Values.lib9cStateService.image.tag | default $.Chart.AppVersion }}"
+        env:
+        - name: RemoteBlockChainStatesEndpoint
+          value: "http://remote-action-evaluator-headless.9c-network.svc.cluster.local:{{ $.Values.remoteActionEvaluatorHeadless.ports.graphql }}/graphql/explorer"
+        imagePullPolicy: Always
+        livenessProbe:
+          initialDelaySeconds: 120
+          periodSeconds: 5
+          successThreshold: 1
+          tcpSocket:
+            port: {{ $.Values.lib9cStateService.ports.http }}
+          timeoutSeconds: 1
+        name: lib9c-state-service
+        ports:
+          - containerPort: 5157
+            name: http
+            protocol: TCP
+        resources:
+          {{- toYaml $.Values.lib9cStateService.resources | nindent 10 }}
+        terminationMessagePath: /dev/termination-log
+        terminationMessagePolicy: File
+      nodeSelector:
+        node.kubernetes.io/instance-type: m5.large
+      dnsPolicy: ClusterFirst
+      restartPolicy: Always
+      schedulerName: default-scheduler
+      terminationGracePeriodSeconds: 30
+      securityContext:
+        {{- toYaml $.Values.lib9cStateService.podSecurityContext | nindent 8 }}
+      {{- with $.Values.lib9cStateService.nodeSelector }}
+      nodeSelector:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
```

Comment:
there are two nodeSelectors

Reply:
I removed the above one

https://github.com/planetarium/9c-infra/compare/5a993f178cd4d204f9d5200a7c520bc2af58298e..13a68b069e244b3c73e4121d300cab84ac800e4a